src/Core_Command.php

```diff
@@ -709,36 +727,42 @@ private static function create_initial_blog( $network_id, $blog_id, $domain, $pa
 		$subdomain_install, $site_user ) {
 		global $wpdb, $current_site, $wp_rewrite;
 
-		$current_site = new stdClass;
-		$current_site->domain = $domain;
-		$current_site->path = $path;
+		$current_site            = new stdClass();
+		$current_site->domain    = $domain;
+		$current_site->path      = $path;
 		$current_site->site_name = ucfirst( $domain );
-		$wpdb->insert( $wpdb->blogs, array(
-			'site_id' => $network_id,
-			'domain' => $domain,
-			'path' => $path,
-			'registered' => current_time( 'mysql' )
-		) );
-		$current_site->blog_id = $blog_id = $wpdb->insert_id;
+		$wpdb->insert(
```

Comment:
Similar to an earlier suggestion, you may want to consider declaring the array as a variable just above and passing it to the function call.